artifacts/testdata/windows/remediation.in.yaml

```diff
@@ -0,0 +1,6 @@
+Queries:
+  # Test Sinkhole remediation - output should be only default artifact entry
+  - SELECT * FROM Artifact.Windows.Remediation.Sinkhole()
+
+  # Test rolling back sinkhole - output none
+  - SELECT * FROM Artifact.Windows.Remediation.Sinkhole(RestoreBackup="True")
```

Comment:
Should we run another query here to ensure the file is rolled back (maybe hash before and after)